README.rst

```diff
@@ -34,11 +50,19 @@ Available Features
 * Charts can be viewed at resolutions of 1 day, 3 days, a week, a month and a year
 * Configurable alerts
 * CSV Export of monitoring data
-* Possibility to `Configure additional charts <#openwisp_monitoring_charts>`_
+* Possibility to configure additional `Metrics <#openwisp_monitoring_metrics>`_ and `Charts <#openwisp_monitoring_charts>`_
 * Extensible active check system: it's possible to write additional checks that
   are run periodically using python classes
 * API to retrieve the chart metrics and status information of each device
```

Comment:
let's mention **NetJSON DeviceMonitoring** and link to http://netjson.org/docs/what.html#devicemonitoring here as well please